keps/sig-docs/20191020-doc-policies-for-third-party-content.md

```diff
@@ -0,0 +1,162 @@
+---
+title: doc-policies-for-third-party-content
+authors:
+  - "@jimangel"
+  - "@sftim"
+  - "@zacharysarah"
+owning-sig: sig-docs
+reviewers:
+  - TBD
+approvers:
+  - TBD
+editor: TBD
+creation-date: 2019-10-20
+last-updated: 2019-10-20
+status: provisional
+---
+
+# doc-policies-for-third-party-content
+
+## Table of Contents
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [User Stories](#user-stories)
+    - [Story 1 (fictional)](#story-1-fictional)
+    - [Story 2 (fictional)](#story-2-fictional)
+    - [Story 3 (actual)](#story-3-actual)
+    - [Story 4 (actual)](#story-4-actual)
+    - [Story 5 (actual)](#story-5-actual)
+  - [Implementation Details/Notes/Constraints](#implementation-detailsnotesconstraints)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Graduation Criteria](#graduation-criteria)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+<!-- /toc -->
+
+*Note* that this KEP does not target any release; SIG Docs follows a continuous release process for website content.
+
+## Summary
+
+Feature docs are not a place for vendor pitches. Nevertheless, SIG Docs has seen pull requests that are more or less a request to place advertising content on the Kubernetes website.
+
+Feature docs also contain content hosted in multiple places (“dual-sourced content”). A good practice for code project docs is to host single-sourced content only, and to provide links to other providers’ single-sourced content.
```

Comment:
not requesting clarification, but just wanted to mention the docker case here.
external websites already provide information on how to install docker, yet we have some further details in our guide:
- the newest docker version is usually not approved yet, thus our guides are usually trailing behind recommending an older / tested docker.
- distros or the docker project bundle a docker version that usually has `cgroupfs` as it's cgroup driver, we recommend changing that to `systemd`.

ref: https://kubernetes.io/docs/setup/production-environment/container-runtimes/#docker

Reply:
Docker (and other 3rd party content tied closely with Kubernetes) will need different treatment from, say, ACME Inc.'s Ingress controller. That's fine.

I'm not sure if there should be a defined list of projects with special treatments, or merely rough consensus and an appreciation that each case can have its own merits.

Reply:
possibly list the exceptions as part of the KEP?